### ISSUE TITLE
Navbar on links(Home,About and Contact)

### ISSUE THREAD

**Issue by Amoghchandragiri** [Amoghchandragiri/Webbit#2]
1) Apply navbar on the links so that they come across as horizontal
2) Apply hovering effect using CSS on links (Home, About and Contact)

**Comment by Rishika631** [comment]
Can I work on this issue please?

**Comment by Amoghchandragiri** [comment]
Of course. You can start working on the issue by referring to the Contribute.MD file.

**Comment by Rishika631** [comment]
Can you assign this issue to me please?

**Comment by Amoghchandragiri** [comment]
Yes, I have already given permission. Start working on it by following the steps from readme.md and contribute.md and generate a pull request. If changes are improving the website, I will merge it
